### DISCUSSION TITLE
how can limit a single licence on a single machine?

### DISCUSSION THREAD

**Discussion by hatembentayeb** [keygen-sh/community#12] [Q&A] [answered]
I m a little bit confused about this, my concern is to make a single license attached to a single machine, prevent the user to share it on another machine, it has to be blocked. 

Is there a way to achieve this? 

Thanks in advance

**Comment by ezekg** [comment, marked as answer]
That's called [a node-locked license](https://keygen.sh/docs/choosing-a-licensing-model/node-locked-licenses/). You would set the policy to have a max machines value of 1. Then you can perform machine activation as you would normally, for a single machine.
